CHANGELOG.md

```diff
@@ -105,6 +105,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * `jj git push` now can push commits with empty descriptions with the
   `--allow-empty-description` flag
 
+* `jj squash` now accepts a `--keep-empty` option to keep the source commit.
```

Comment:
I'll just leave this here as a reminder to sync up the commit description and contents before merging (after the option name is finalized). Currently, it's `--keep-emptied` in the name and `--keep-empty` in the contents.

Reply:
And `jj rebase` has `--skip-empty` with "skip emptied" behavior. If we prefer the `-emptied` name because it's more correct, then we should also update `rebase` (in a separate PR).